package.json

```diff
@@ -21,6 +21,7 @@
 		"test-compile": "nyc mocha --timeout 0 --recursive out/CLI/test.js",
 		"test-rojo": "rojo build tests -o ./tests/test.rbxlx",
 		"test-run": "lune ./tests/test.lua ./tests/test.rbxlx",
+		"test-roblox": "npm run build && npm run test-compile && npm run test-rojo && npm run run-in-roblox",
```

Comment:
Would it be preferable to swap these around? `npm test` will be the intuitive command for local development and should use the best-available testing suite, that being run-in-roblox.